src/Select/selectStyles.ts

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled';
+import { lightenDarkenColor } from 'src/styles/stylesUtil';
+import { GroupTypeBase, Styles } from 'react-select';
+import { Theme } from '@emotion/react';
+import { SelectOption } from 'src/types/global';
+import { ReactComponent as baselineExpandSvg } from './baseline-expand_more-24px.svg'
+const DARKEN_MOST = -40;
```

Comment:
Are these defined elsewhere for the buttons? Maybe they can be added to the theme or exported from a more global folder file if they're used in more places.